oaknational/Oak-Web-Application
# chore: update exam board copy CUR-297

## Description

- [ ]  Text “All sequences are identical at KS3” is shown in subject phase picker where there are exam-board choices available
- [ ]  The text is not shown where it is not

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1983--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-primary/overview
2. Click on _______
3. You should see _______
<img width="1177" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/7987787/004cbe9f-8082-4bee-944b-823fc12d3d68">

## Checklist
